apps/web/pages/event-types/index.tsx

```diff
@@ -487,29 +500,28 @@ const CreateFirstEventTypeView = ({ canAddEvents, profiles }: CreateEventTypePro
   );
 };
 
+const CTA = () => {
+  const query = trpc.useQuery(["viewer.eventTypes"]);
+
+  if (!query.data) return null;
+
+  return (
+    <CreateEventTypeButton canAddEvents={query.data.viewer.canAddEvents} options={query.data.profiles} />
+  );
+};
```

Comment:
Created a separate component to colocate state here. Otherwise, it busts the optimization benefit we get with `withQuery`. react-query is smart to dedupe queries so there's no concern for refetching.